apps/bbsync/save.py

```diff
@@ -129,10 +137,17 @@ def flaw(self):
     @property
     def model(self):
         """
-        Flaw model class getter
+        instance model class getter
```

Comment:
Nitpick: since this is already a concrete BugzillaSaver for Flaws, wording `Flaw model class getter` might stay.

Reply:
Good point, I will rename back